core/server/master/src/main/java/alluxio/master/file/meta/InodeLockManager.java

```diff
@@ -159,13 +160,13 @@ public LockResource lockEdge(Edge edge, LockMode mode) {
   }
 
   /**
-   * Acquires the lock for modifying an inodes last modified time.
+   * Acquires the lock for modifying an inodes last modified time or size.
    *
    * @param inodeId the id of the inode to lock
    * @return a lock resource which must be closed to release the lock
    */
-  public LockResource lockLastModified(long inodeId) {
-    return new LockResource(mLastModifiedLocks.get(inodeId));
+  public LockResource lockParentUpdate(long inodeId) {
```

Comment:
is "parent" confusing? It could mean the parent of the id we specify. Why not something like "lockUpdate", or "writeLock", etc?

Reply:
Changed to `lockUpdate`